etc/bash_completion.d/brew

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
I don’t think we need this line because the file is sourced rather than executed.